kolibri/core/assets/src/views/immersive-full-screen/index.vue

```diff
@@ -85,6 +85,6 @@
     top: 0
     bottom: 0
     overflow-y: auto
-    background-color: $core-bg-canvas
+    background-color: $core-bg-light
```

Comment:
@khangmach any issues with the immersive full screen background being changed to the `light` colour, rather than the `canvas` colour?

Reply:
no prob!